run_mpas_analysis

```diff
@@ -31,7 +31,7 @@ from mpas_analysis.shared.html import generate_html
 from mpas_analysis.shared import AnalysisTask
 
 
-def build_analysis_list(config):  # {{{
+def build_analysis_list(config, refConfig):  # {{{
```

Comment:
@xylar: is this the best place where we could add a check on the refConfig file? something on whether the file exists for example. Also, something that says, if refConfig is defined: 'Comparing to reference run rather than observations. Make sure to have run MPAS-Analysis on the ref config file previously', or something along these lines. Thought this would be a nice user-friendly addition, if you agree.

Reply:
The right place is a little higher up.  I have added both the requested check and the requested print statement.